libpod/container_exec.go

```diff
@@ -848,17 +848,16 @@ func (c *Container) healthCheckExec(config *ExecConfig, timeout time.Duration, s
 		return -1, err
 	}
 
-	if c.state.ExecSessions == nil {
-		c.state.ExecSessions = make(map[string]*ExecSession)
-	}
-	c.state.ExecSessions[session.ID()] = session
-	defer delete(c.state.ExecSessions, session.ID())
-
```

Comment:
do we leak the session if podman crashes in healthCheckExec?

Reply:
am I missing something, I thought the point was already the we do not commit the session to the db https://github.com/containers/podman/commit/ad9839ac555865a81a7adae62a9a8be1f2c2268f?